example/gardener-local/calico/base/kustomization.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://raw.githubusercontent.com/projectcalico/calico/v3.25.0/manifests/calico.yaml
+- https://raw.githubusercontent.com/projectcalico/calico/v3.26.1/manifests/calico.yaml
 
 images:
 - name: docker.io/calico/cni
-  newName: eu.gcr.io/gardener-project/3rd/calico/cni
+  newName: quay.io/calico/cni
 - name: docker.io/calico/node
-  newName: eu.gcr.io/gardener-project/3rd/calico/node
+  newName: quay.io/calico/node
 - name: docker.io/calico/kube-controllers
-  newName: eu.gcr.io/gardener-project/3rd/calico/kube-controllers
+  newName: quay.io/calico/kube-controllers
```

Comment:
Why we need to overwrite them? The quay.io images are the default images. So I would suggest to use the default from the networking-calico extension. I assume that such overwrite was introduced as back then maybe calico was using the dockerhub images.
cc @timebertt